src/server/server.ts

```diff
@@ -1,18 +1,8 @@
 import * as WebSocket from 'ws';
 
-import {
-  SUBSCRIPTION_FAIL,
-  SUBSCRIPTION_DATA,
-  SUBSCRIPTION_START,
-  SUBSCRIPTION_END,
-  SUBSCRIPTION_SUCCESS,
-  KEEPALIVE,
-  INIT,
-  INIT_FAIL,
-  INIT_SUCCESS,
-} from './messageTypes';
-import {GRAPHQL_SUBSCRIPTIONS} from './protocols';
-import {SubscriptionManager} from 'graphql-subscriptions';
+import MessageTypes from '../common/message-types';
+import { GRAPHQL_WS } from '../common/protocol';
+import { SubscriptionManager } from 'graphql-subscriptions';
```

Comment:
hmm, i think we should convert subscription manager into an interface here and not import graphql-subscriptions,
doesn't make much sense that the transport layer depended on the "entity" that need's it's service..